fuse/data/ops/hashable_class.py

```diff
@@ -0,0 +1,65 @@
+from typing import Dict, Union, List, Sequence, Any, Optional, Callable
+from fuse.data.ops import get_function_call_str
+from inspect import stack
+from fuse.data.ops.caching_tools import get_callers_string_description, value_to_string
+
+class HashableClass:
```

Comment:
Theoretically, we can move HashableClass, get_callers_string_description, and value_to_string to utils and to use it in other cases as well.
Leave it to you to decide.